code/modules/antagonists/eldritch_cult/knowledge/flesh_lore.dm

```diff
@@ -214,7 +214,7 @@
 	gain_text = "Man of this world. Hear me! For the time of the lord of arms has come! Emperor of Flesh guides my army!"
 	desc = "Bring 3 bodies onto a transmutation rune to gain the ability of shedding your human form, and gaining untold power."
 	required_atoms = list(/mob/living/carbon/human)
-	cost = 3
+	cost = 9
```

Comment:
I do believe this is intentional that its 9 instead of 12, but i could be wrong

Reply:
No I think that's my mistake.